x-pack/plugins/timelines/public/components/utils/keury/index.ts

```diff
@@ -74,20 +74,24 @@ export const convertToBuildEsQuery = ({
   indexPattern: DataViewBase;
   queries: Query[];
   filters: Filter[];
-}) => {
+}): [string, undefined] | [undefined, Error] => {
```

Comment:
The type returned from this (`timelines` plugin) implementation of `convertToBuildEsQuery` is now the same as the type originally defined by the Security Solution in `x-pack/plugins/security_solution/public/common/lib/keury/index.ts`. (That implementation is deleted in this PR, so we are left with just one implementation of `convertToBuildEsQuery`.)